lab_5_scrapper/scrapper.py

```diff
@@ -1,82 +1,149 @@
 """
 Crawler implementation
 """
+import datetime
+import json
+from pathlib import Path
 from typing import Pattern, Union
+from urllib.parse import urlparse
 
+import requests
+from bs4 import BeautifulSoup
+
+from core_utils.article.article import Article
+from core_utils.article.io import to_meta, to_raw
+from core_utils.config_dto import ConfigDTO
+from core_utils.constants import (ASSETS_PATH, CRAWLER_CONFIG_PATH,
+                                  NUM_ARTICLES_UPPER_LIMIT,
+                                  TIMEOUT_LOWER_LIMIT, TIMEOUT_UPPER_LIMIT)
+
+class IncorrectSeedURLError(Exception):
+pass
+
+
+class NumberOfArticlesOutOfRangeError(Exception):
+pass
+
+class IncorrectNumberOfArticlesError(Exception):
+pass
+
+class IncorrectHeadersError(Exception):
+pass
+
+class IncorrectEncodingError(Exception):
+pass
+
+class IncorrectTimeoutError(Exception):
+pass
+
+class IncorrectVerifyError(Exception):
+pass
 
 class Config:
     """
     Unpacks and validates configurations
     """
+    seed_urls: list[str]
+    total_articles_to_find_and_parse: int
+    headers: dict[str, str]
+    encoding: str
+    timeout: int
+    verify_certificate: bool
+    headless_mode: bool
 
     def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self._validate_config_content()
+        configDTO = self._extract_config_content()
```

Comment:
please refer to python variables naming style and fix this accordingly

Reply:
fixed